composer.json

```diff
@@ -39,7 +39,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpspec/phpspec": "^4.2 || ^5.0",
+        "phpspec/phpspec": "^4.2 || ^5.0 || ^6.0",
```

Comment:
phpspec 6 requires at least PHP 7.2.

Maybe we should release a new major version (with phpspec v6 support) and also update our requirement to `"php": "^7.2"`. :thinking:

Reply:
Yes I knew that phpspec require 7.2 but for BC break I still require 7.1

Reply:
Hello !

It's not really a problem here since we are just "allowing" the usage of phpspec 6.0.

Since we are using `^7.1` for the php version constraint, it allow using PHP7.2 without any problem. Then depending on the environment and the project context it will be the composer resolution algorithm that must trigger errors or warnings.